WordPress/twentytwentyone
# Dark Mode: Remove toggle code from the editor

See #846 — the dark mode toggle is still being injected into the editor, even though it was intentionally removed. This PR removes the code related to injected the toggle button into the editor.

## Relevant technical choices:
Removes the JS file used in the editor, the ajax action that returns the HTML, and the style output function (which was only used in the ajax action). It also removes the `display: none` since the button doesn't exist anymore.

## Test instructions

This PR can be tested by following these steps:
1. Open the editor
2. There should be no toggle button
3. Inspect element, look for the div with `editor-styles-wrapper`; the first child item should not be a button
4. Switch back to the frontend
5. That toggle should still work as expected

## Quality assurance
